extras/rya.geoindexing/src/test/java/org/apache/rya/indexing/geotemporal/MongoGeoTemporalIndexIT.java

```diff
@@ -0,0 +1,186 @@
+        final Optional<Event> event = events.get(subject);
+        assertTrue(event.isPresent());
+
+        indexer.close();
+    }
+
+    @Test
+    public void constantSubjQuery_Test() throws Exception {
+        addStatements();
+        final String query =
+                "PREFIX time: <http://www.w3.org/2006/time#> \n"
+              + "PREFIX tempo: <tag:rya-rdf.org,2015:temporal#> \n"
+              + "PREFIX geo: <http://www.opengis.net/ont/geosparql#>"
+              + "PREFIX geof: <http://www.opengis.net/def/function/geosparql/>"
+              + "SELECT * "
+              + "WHERE { "
+                + "  <urn:event1> time:atTime ?time . "
+                + "  <urn:event1> geo:asWKT ?point . "
+                + "  FILTER(geof:sfWithin(?point, \"POLYGON((-3 -2, -3 2, 1 2, 1 -2, -3 -2))\"^^geo:wktLiteral)) "
+                + "  FILTER(tempo:equals(?time, \"2015-12-30T12:00:00Z\")) "
+              + "}";
+
+        final TupleQueryResult rez = conn.prepareTupleQuery(QueryLanguage.SPARQL, query).evaluate();
+        final Set<BindingSet> results = new HashSet<>();
+        while(rez.hasNext()) {
+            final BindingSet bs = rez.next();
+            results.add(bs);
+        }
+        final MapBindingSet expected = new MapBindingSet();
+        expected.addBinding("point", VF.createLiteral("POINT (0 0)"));
+        expected.addBinding("time", VF.createLiteral("2015-12-30T07:00:00-05:00"));
+
+        assertEquals(1, results.size());
+        assertEquals(expected, results.iterator().next());
+    }
+
+    @Test
+    public void variableSubjQuery_Test() throws Exception {
+        addStatements();
+        final String query =
+                "PREFIX time: <http://www.w3.org/2006/time#> \n"
+              + "PREFIX tempo: <tag:rya-rdf.org,2015:temporal#> \n"
+              + "PREFIX geo: <http://www.opengis.net/ont/geosparql#>"
+              + "PREFIX geof: <http://www.opengis.net/def/function/geosparql/>"
+              + "SELECT * "
+              + "WHERE { "
+                + "  ?subj time:atTime ?time . "
+                + "  ?subj geo:asWKT ?point . "
+                + "  FILTER(geof:sfWithin(?point, \"POLYGON((-3 -2, -3 2, 1 2, 1 -2, -3 -2))\"^^geo:wktLiteral)) "
+                + "  FILTER(tempo:equals(?time, \"2015-12-30T12:00:00Z\")) "
+              + "}";
+
+        final TupleQueryResult rez = conn.prepareTupleQuery(QueryLanguage.SPARQL, query).evaluate();
+        final List<BindingSet> results = new ArrayList<>();
+        while(rez.hasNext()) {
+            final BindingSet bs = rez.next();
+            results.add(bs);
+        }
+        final MapBindingSet expected1 = new MapBindingSet();
+        expected1.addBinding("point", VF.createLiteral("POINT (0 0)"));
+        expected1.addBinding("time", VF.createLiteral("2015-12-30T07:00:00-05:00"));
+
+        final MapBindingSet expected2 = new MapBindingSet();
+        expected2.addBinding("point", VF.createLiteral("POINT (1 1)"));
```

Comment:
Might be a bug: point 1,1 is entirely on the edge of the polygon in the select filter.  By the definition of SFWithin, it should be excluded.  That is what GeoMesa does.  I discovered while testing the accumulo version of this test.